devtools/ci/travis/install.sh

```diff
@@ -35,13 +35,13 @@ else # Otherwise, CPython... go through conda
         conda install -y -n myenv boost==1.59.0 -c omnia
         conda install -y -n myenv nglview -c bioconda
         conda install -y ambertools=17.0.0 -c http://ambermd.org/downloads/ambertools-rc/conda/
-        python -c "import sander; print(sander)"
-        python -c "import parmed; print(parmed)"
     else
         # Do not install the full numpy/scipy stack
         conda create -y -n myenv python=$PYTHON_VERSION numpy nose pyflakes=1.0.0 \
             coverage nose-timer python-coveralls
     fi
 
     source activate myenv
+    python -c "import sander; print(sander)"
+    python -c "import parmed; print(parmed)"
```

Comment:
for my testing. For some reasons sander is not imported properly.

(You might want to turn off notification for this thread :D).